integration-tests/src/test/resources/features/webhookaction.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Webhook Action tests
+
+  @webhookaction
+  Scenario: Webhook is correctly called
+    Given authenticate against Manager
+    And create a new Bridge "mybridge"
+    And the Bridge "mybridge" is existing with status "ready" within 2 minutes
```

Comment:
```suggestion
    And the Bridge "mybridge" is existing with status "ready" within 4 minutes
```

Reply:
@sutaakar could you please clarify, I see 2 minutes also in action-slack.feature. Should I increase the timeout also there?

Reply:
will do